src/app.d.ts

```diff
@@ -2,6 +2,15 @@
 // for information about these interfaces
 declare global {
     namespace App {
+        interface LinkList {
```

Comment:
Because we're using Typescript we have to declare our own types here to ensure the svelte app knows about them.